### DISCUSSION TITLE
instantiate hub with setting axis not possible

### DISCUSSION THREAD

**Discussion by JohnGottiGe** [pybricks/support#1396] [General]
Hi Everyone,
I just installed pybricks on our inventor hub and since I want to use the inbuild PID to drive and we have our hub installed horizontally, with the right side as the front. So I have to change the axis during instansiation of the hub. So I tried myHub = PrimeHub(top_side=Axis.Z, front_side=Axis.Y), but this did not work, I get the error that Axis isn't defined. 
Can somebody help with this issue?

**Comment by laurensvalk** [comment]
Here is an example you could try: https://docs.pybricks.com/en/latest/hubs/primehub.html#using-a-custom-hub-orientation

Does your script include the following line?

```python
from pybricks.parameters import Axis
```

**Reply by JohnGottiGe** [reply]
Hi Laurens,
thank you very much. What a shame, I did not import the Axis and that's why Axis were unknown :-(
Thanks a lot
John

**Reply by laurensvalk** [reply]
It’s a good way to get familiar with Python error messages 😄 

They’re a bit cryptic at first, but can be quite useful sometimes.
